primal-bit/src/lib.rs

```diff
@@ -10,11 +10,15 @@
 
 //! A very simple bit-vector that serves the needs of `primal`.
 
+#![cfg_attr(feature = "no-std", no_std)]
```

Comment:
I would prefer to make this unconditional in all crates, and then use `cfg` only on the places that need to choose std or libm. I think in _this_ case, `primal-bit` doesn't actually need anything from `std` at all, so we don't even need a feature here.
```suggestion
#![no_std]
```

Reply:
I can't mark reviews as resolved but this has been fixed.

Reply:
@piotrfila could you mark the outdated reviews as resolved?